tests/tensorflow/test_tensorflow2_model_export.py

```diff
@@ -814,3 +815,16 @@ def test_log_model_with_code_paths(saved_tf_iris_model):
         _compare_logged_code_paths(__file__, model_uri, mlflow.tensorflow.FLAVOR_NAME)
         mlflow.tensorflow.load_model(model_uri)
         add_mock.assert_called()
+
+
+def test_saved_model_support_array_type_input():
+    def infer(features):
+        res = np.expand_dims(features.numpy().sum(axis=1), axis=1)
+        return {"prediction": tf.constant(res)}
+
+    model = _TF2Wrapper(None, infer)
+    infer_df = pd.DataFrame({"features": [[1.0, 2.0, 3.0, 4.0], [5.0, 6.0, 7.0, 8.0]]})
+
+    result = model.predict(infer_df)
+
+    np.testing.assert_allclose(result["prediction"], [10.0, 26.0])
```

Comment:
Where does `[10.0, 26.0]` come from?

Reply:
Ah I see

Reply:
```suggestion
    np.testing.assert_allclose(result["prediction"], infer_df.<method to compute sum>)
```

Can we replace it with `infer_df.<method to compute sum>`?